netctlsystraymenu@prmurthy/files/netctlsystraymenu@prmurthy/applet.js

```diff
@@ -257,19 +263,19 @@ Netctl.prototype = {
 
         if (iface == "") {
             if (contype == "A") {
-                msg = "The wireless network" + newprofileName + " is not available. Re-enabling the auto profile. Please enter your password again";
+                msg = _("The wireless network") + newprofileName + _(" is not available. Re-enabling the auto profile. Please enter your password again");
```

Comment:
This can be better done with %s and .format, e.g.
https://github.com/linuxmint/cinnamon-spices-applets/search?l=JavaScript&p=2&q=%25s&type=&utf8=%E2%9C%93

But needs to be tested.